pam/integration-tests/testdata/TestCLIIntegration/golden/authenticate_user_switching_username

```diff
@@ -231,8 +231,8 @@ Acct mgmt return: <nil>
 ────────────────────────────────────────────────────────────────────────────────
 > ./pam_authd socket=/tmp/pam-cli-tests.sock
 Gimme your password
-Auth return: <nil>
-Acct mgmt return: <nil>
+Auth return: Authentication failure: user not found
```

Comment:
are those expected?

Reply:
Nope, they are not. See my previous comment about the fluke :confounded: